### ISSUE TITLE
Call for a more durable approach

### ISSUE THREAD

**Issue by hellomedia** [vshymanskyy/StandWithUkraine#173]
I originally posted this in another repo, but this is probably the right place for it.

I understand your initiative comes from a place of good intentions. However, I argue that the view it propagates is unhelpful to making the world a better place.

To avoid future wars, the world needs to encourage more cross-culture understanding, and to stay away from the good vs evil narrative.

I am afraid this type of initiative does the opposite.

It suggests that propaganda exists only for others, which is a dangerous illusion. If this claim seems strange to you, please re-read "Manufacturing Consent" or read about Edward Bernays, the father of modern day propaganda, who spent his life successfully putting his theory to practice to misinform and manipulate public opinion in the US.

It implies that there are good guys and bad guys. This ignores the complexity of the history and geopolitics of the region, as well as in the world between the US and Russia. If this claim seems strange to you, catch some interventions from Jeffrey Sachs, a guy who has devoted his life to tackling the biggest challenges of mankind (poverty, climate change, UN sustainability goals) and has worked as advisor for a number of governments around the globe, and as such, has a unique insight on how political and cultural narratives are shaped in various parts of the world.

It is important to realize that, no matter the crime, no court of Justice would label one side "good" and the other side "bad". Nor would it ever refuse to look at context. No matter the crime, time is spent listening to the defense arguments. These arguments are presented by the defense itself, not by anyone else. These are the requirements to approach Justice.

In our collective trial of the Ukraine war, we do not do any of that. We refuse to look at context. We refuse to hear the defense arguments. The only analysis we hear is from our very own media, the one ingrained in our narrative; whose modus operandi is to demonize and delegitimize the opposite narrative.

I argue that these elements are the building blocks of a simplistic worldview, which is precisely at the root of tensions causing wars. 

And that the world will only be a better place when we start taking a different approach.

------------

When I shared [this in the other repo](https://github.com/Ocramius/ProxyManager/commit/281212c17a185c6dabcb4ed1e1cd4962941b0165#commitcomment-87939923), I got this answer :

> No, people are being bombed, killed and deported by a fascist country (Russia).
> 
> That's it.
> 
> Ukraine is defending itself from an invasion, end of story.
>
> Also, you are now blocked from my repositories.

This is symptomatic of the problem.

You do not have an informed opinion if you block anything you don't like.

You do not make the world a better place by cancelling someone trying to engage with you in a respectful way.

You only fuel more frustrations, resentment and hatred.

----------

Hopefully, this can be read by people who can stay away from knee jerk reaction, take it with them as food for thought, and maybe, over time, make the world a better place.

**Comment by vshymanskyy** [comment]
My suggestion is to visit Ukraine. You'll be able to build your own opinion pretty quickly.

**Comment by hellomedia** [comment]
Thank you for your response.

You are right, if I had experienced  the destruction and horror in Ukraine first hand (which I obviously don't deny in case there was any misunderstanding about that), I certainly would feel differently.

But that's precisely the point. In a court of Law, Justice is not decided by victims.

If you want to make the world a better place in the long run, you have to aim for Justice. For that, you have to take emotions out of the equation and take a hard look at context.

In your repo contributors, how many might be fueled by a sense of revenge ?

**Comment by vshymanskyy** [comment]
> Justice is not decided by victims.

yes, but what is the responsibility for unrecoverable damage, like mass-murdering?

Russia is doing that over and over again throughout decades, yet I didn't face justice yet. I think time has come.

**Comment by PlutoCA** [comment]
A instigated b to attack c, and c taught b a lesson for self-preservation. see the reason behind it rationally. No one wants to start a war, Russia also agreed to peace talks as long as Ukraine does not join NATO, your actor president did not agree just

**Comment by hellomedia** [comment]
> yes, but what is the responsibility for unrecoverable damage, like mass murdering?

That is for a court of justice to decide.

If you want my 2 cents, your country should get reparations from a large number of countries (including, but not limited to Russia) responsible for the destruction of Ukraine. A fair, truly independent court (not just headed by the west) should decide what the responsibilities are, after a careful look at the causes of the conflict.

This is a genuine question. Have you watched John Mearsheimer's arguments about the responsibilities of the war ?
If so, what do you make of them ?

> Russia has been doing that repeatedly for decades, yet it didn't face justice yet. I think the time has come.

Sure. I am all about justice being rendered.

I am still concerned about the long term real life implications of this initiative.

At the moment, I see the goal as "spreading the word". Is that correct ?

**Comment by vshymanskyy** [comment]
1. spreading the word
2. collecting donations for Ukraine
3. negatively impacting Russia's ability to escalate

**Comment by webknjaz** [comment]
@PlutoCA you demonstrate zero understanding of the context and seem to be clueless history-wise with such statements. r*ssia never keeps the word. Not now, not in the past. It sometimes pretends to do so for short periods of time to regroup and organize even worse atrocities.

@hellomedia please, don't join the westsplaining efforts without trying to understand the context. You yourself have just used several propagandist points spread by the ruscist terrorists (like the victim-blaming by not calling out the aggressor and naming the war an “Ukraine war” as if it's happening by itself, or hinting it being a “proxy” war — this all indicates either zero understanding of the context or just the support of the terrorist propaganda, or both). Don't assume that the authors of this project don't know what they are talking about. All of us are Ukrainians. All of us know our history very well and pretty much all of us have family stories of surviving previous genocides organized solely by the r*ssians, most Ukrainians have those stories. This isn't something you get learn from a history book in full — there might be some mentions but not the full diversity of the ussr occupation, for example.
Currently, your arguments simply read as “if you could only stop inconveniencing the 'not an agressor' by wanting to live your life, you wouldn't be responsible for them wanting to commit genocides”. This is only harmful to Ukraine, but is quite beneficial to the actual genocidal maniacs.

**Comment by cawa-93** [comment]
Dear @hellomedia, if some day someone invade to your house, kill your wife, rape your chilldren Would you say that the attacker is a "Bad Man"? Context? The attacker wanted to fuck. Perhaps it was your children who provoked him. Maybe they wore too revealing clothes one day in the park and thereby made the attacker want to fuck. Does that make a person not a "bad guy"? After all, he wanted to fuck. It would be only fair if he had the opportunity to satisfy his desires, wouldn't it? That doesn't make him a "bad guy," does it? Maybe he is just a victim of circumstances? And your wife should go to a dialogue with him and settle this conflict peacefully. Make concessions to each other. Then she would not have died, her husband would have been fucked, and one of your children might even have remained unharmed. But your wife attacked the assailant and he was forced to kill her. It is not fair that anyone should suffer like this. You should shake his hand and leave peacefully. After all, he is not a "bad person". You are not a "good person" either. All this is just a conflict of interest.

**Comment by AurimasNav** [comment]
> A instigated b to attack c, and c taught b a lesson for self-preservation. see the reason behind it rationally. No one wants to start a war, Russia also agreed to peace talks as long as Ukraine does not join NATO, your actor president did not agree just

Why should russia or putin have any say in what Ukraine decides to join? By that logic it would be justified for NATO to invade Ukraine if she decided to join CSTO?

**Comment by hellomedia** [comment]
@vshymanskyy You did not address the concerns expressed in this issue.

Let me rephrase them.

Your initiative is promoting a western worldview. It blocks everything that does not fit the western narrative. I asked you if you have heard John Mearsheimer's arguments about the responsibilities of the war, you did not answer. He is not a Russian propagandist. He spent his life analyzing geopolitics with a strong pro American perspective. I could point you to Jeffrey Sach's arguments as well. He is a man with unique international geopolitical experience, who knows a thing or two from the geopolitcs of the region. He has served as economic advisor for Poland, Slovenia, Estonia, Gorbatchev and Elstine of Russia after the collapse of the Soviet Union, and president Kouchma of Ukraine in the first year of its independance . He has heard the different perspectives. He also served as special advisor to 3 UN secretary generals, Ban Ki Moon, Kofi Anan and Antonio Gutierres working on poverty alleviation, health and aid policy, sustainability, etc. Not your average dude. Probably worth listening to what he has to say. 

Merheimer: https://www.youtube.com/watch?v=qciVozNtCDM
Sachs : https://youtu.be/NnnRXZnAtF0?t=1025

By not answering on those points, I am getting the impression you refuse to look at anything that does not fit your narrative. 

That is problematic because refusing to listen to different views fuels polarization, which fuels tensions, which leads to wars.

So in your initiative I see victims making justice themselves, spreading a western narrative that is only a part of the truth, which in the end fuels frustrations and hatred outside of the western bubble, making the world a more dangerous place in the long run.

So my question is, would there be a way to help Ukrainian people while showing more respect towards the non-western world which sees the hypocrisy and double standards propagated by the western narrative ?

**Comment by vshymanskyy** [comment]
> which sees the hypocrisy and double standards propagated by the western narrative ?

What exactly (in this repo) are you referring to? please pin-point.

**Comment by hellomedia** [comment]
@webknjaz I am sorry, I don't seem to agree with some of your points. Nothing personal, but if we are having a discussion, I have to explain my views.

> All of us know our history very well and pretty much all of us have family stories of surviving previous genocides organized solely by the r*ssians, most Ukrainians have those stories.

I believe you when you say you all have horrific stories of things comited by Russians. Every war is fed by horrific stories about the enemy. Some of them are true, some of them are not. It's always been like that, I don't see why it would be different in Ukraine.

>  You yourself have just used several propagandist points spread by the ruscist terrorists

Stating that something is false simply because it is claimed by russian propaganda is not a valid argument. Propaganda uses everything that fits its goal - some false, some true. If they say the sky is blue, I will agree.

> Currently, your arguments simply read as “if you could only stop inconveniencing the 'not an agressor' by wanting to live your life, you wouldn't be responsible for them wanting to commit genocides”. This is only harmful to Ukraine, but is quite beneficial to the actual genocidal maniacs.

My argument is : 

I just don't think black/white good/evil worldviews are helpful. I support any initiative to help and get peace to the Ukrainian people, but not at the cost of doing more harm than good.

That's all. I don't deny Russia has a responsibility, I just don't know the extent of that, compared to the responsibility of the US. I repeat, I don't deny Russia has responsibility ( any time the responsibility of the US is mentioned, it gets taken as a exoneration of Russia. That is childish). The world is complex. If we want Justice, we have to take a hard look at things, and not be satisfied with a quick blame game like 10 years old.

> like the victim-blaming by not calling out the aggressor

Of course Russia is the aggressor. But that word is too final. It blocks all analysis. Refusing to analyse things is never good. It makes it impossible to find solutions. So when it's used with the goal of blocking all analysis - as it often is - it is a poor choice of word. Invader would be a better word.

>  and naming the war an “Ukraine war” as if it's happening by itself

Vietnam war, Iraq war, Korean war.... Those are names used all the time. In my mind, it's just geographical. But you are probably right, I will try to use "war in Ukraine" next time (I suppose that's better? it's just geographical)

But if you want to call it "Putin war", I have something to say about that. Since when did we start calling wars from the name of the invader ? Did we ever call the Iraq war the "Bush war" ? Was WWII called the "Hitler war" ? This double standards is western propaganda in action, it is used in the media, and people are blind to it.

> or hinting it being a “proxy” war — this all indicates either zero understanding of the context or just the support of the terrorist propaganda, or both)

You don't think the US is waging a proxy war ? It's so obvious even western media starts admitting it.


I suppose we just have to agree to disagree on those points. I hope you will consider that just because I don't agree, it doesn't make me an enemy of your country or a "Russian nazi". I wish the best to you and your family in those tough times, and peace to Ukraine.

**Comment by vshymanskyy** [comment]
I see no point in this discussion. If you want to find truth, visit
Ukraine. Or Russia.

**Comment by hellomedia** [comment]
Your intiative is contributing to make the world a more dangerous place. Live with that then.

**Comment by vshymanskyy** [comment]
When I asked you to pin-point where in this repo this is happening, I didn't get a reply.

**Comment by hellomedia** [comment]
I will if you give me time to do it. Tomorrow.

You didn't reply to my questions.

**Comment by vshymanskyy** [comment]
It's ok, take your time. Without such inputs, it's hard to be constructive while providing my answers :)

**Comment by hellomedia** [comment]
Fair enough. Thank you for focusing the debate on the constructive things.

**Comment by Kratacoa** [comment]
@hellomedia the points you make are not new to any Ukrainian that have confronted themselves with woke westerners (or other less innocent groups), and frankly, they are irritating at best, and infuriating at worst.
Starting this thread and the way you frame this discussion are inherently futile: the moment you decide to face it with a skeptic whataboutist attitude and assume moral superiority (thanks to the way you build your narrative), you are doomed to not have any result that involves taking action.
I could argue in more detail, but I hope you spare everyone the pain and take a moment to reflect on whether it is appropriate to engage in west-splaining for what is a painful argument for everyone involved here. Frankly, there is no need for any theoretical framework or ideology, they all evaporate in front of the brutal reality of what is happening in war scenario: the moral choice is obvious.
